.devcontainer/toolchain-cpp.dockerfile

```
@@ -74,4 +75,10 @@ RUN . /root/toolchain.env \
 RUN sudo apt-get update && sudo apt-get install -y python3-pip \
     && sudo pip3 install --no-cache-dir gcovr --break-system-packages \
     && sudo apt-get clean \
-    && sudo rm -rf /var/cache/apt/archives/ /var/lib/apt/lists/*
\ No newline at end of file
+    && sudo rm -rf /var/cache/apt/archives/ /var/lib/apt/lists/*
+
+# Build OpenAPI
+RUN echo "INSTALLING OPENAPI"
+COPY smpte /root/smpte
```

```suggestion
COPY smpte /root/smpte/install-tooling.sh
```
So that only the script is left in the image